ui/lib/kv/addon/components/page/list.js

```diff
@@ -24,13 +25,17 @@ export default class KvListPageComponent extends Component {
   @service flashMessages;
   @service router;
 
-  @tracked secretPath = '';
+  @tracked secretPath;
 
   get mountPoint() {
     // mountPoint tells the LinkedBlock component where to start the transition. In this case, mountPoint will always be vault.cluster.secrets.backend.kv.
     return getOwner(this).mountPoint;
   }
 
+  get buttonText() {
```

Comment:
Maybe we also move this to the `input-search` component since it's related to the tracked value in that child?